packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/create-sign-jwt/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Create and sign the JWT
+---
+
+> **Note:** Okta SDKs support creating and signing the JWT and requesting an access token. If you are using an Okta SDK, you can skip this section and the <GuideLink link="../get-access-token">Get an access token</GuideLink> section.
+
+Create and sign the JWT with your private key for use as a JWT assertion in the request for a scoped access token. You can create this `client_credentials` JWT in several ways.
+
+For testing purposes, use [this tool](https://www.jsonwebtoken.dev/) to generate and sign a JWT. This tool supports both JWT and PEM formats. For a production use case, see [Build a JWT with a private key](/docs/guides/build-self-signed-jwt/java/jwt-with-private-key/) for both a Java and a JavaScript example of signing the JWT.
+
+> **Note:** After the service app has Okta-scoped grants, only an admin with Super Admin permissions can rotate the keys.
+
+The following [JWT claims](/docs/reference/api/oidc/#token-claims-for-client-authentication-with-client-secret-or-private-key-jwt) can be used in the request for a scoped access token:
+
+* `aud` &mdash; The full URL of the resource that you're using the JWT to authenticate to
+* `exp` &mdash; The expiration time of the token in seconds since January 1, 1970 UTC (current UNIX timestamp). This value must be a maximum of only an hour in the future.
```

Comment:
Isn't `exp` also an optional parameter?

Reply:
no, it's required: https://developer.okta.com/docs/reference/api/oidc/#token-claims-for-client-authentication-with-client-secret-or-private-key-jwt

Reply:
If so, in the example you provided below, you don't have `exp` in the payload. Am I missing something? 🤔